pkg/config/parameters_test.go

```diff
@@ -126,6 +126,24 @@ func TestValidateAccessLogType(t *testing.T) {
 	assert.NoError(t, JSONAccessLog.Validate())
 }
 
+func TestValidateAccessLogFields(t *testing.T) {
+	assert.Error(t, AccessLogFields([]string{"dog", "cat"}).Validate())
```

Comment:
Should we accept something like`[]string{"dog=pug", "cat=black"}`?

Reply:
@jpeach in theory Envoy supports that and it will put the static value `dog: field` in access log.  I think, if we're supporting other string operators (static strings) we should also accept this.